src/subscription/alerts/ErrorMessages.jsx

```diff
@@ -1,6 +1,49 @@
 // import FallbackErrorMessage from '../../feedback/FallbackErrorMessage';
+import React from 'react';
+import { getConfig } from '@edx/frontend-platform';
+import { Hyperlink } from '@edx/paragon';
+import { FormattedMessage } from '@edx/frontend-platform/i18n';
+
 // eslint-disable-next-line import/prefer-default-export
-export const EmptySubscriptionMessage = () => (
-  <>
-  </>
+export const EmbargoErrorMessage = () => (
```

Comment:
Are we sure that for subscription programs we want to show embargo error this way?
Currently in ecommerce we are just showing fallback error. I might be missing something here if we have decided to do it this way now for subscriptions.